### ISSUE TITLE
Better error message for duplicate labels

### ISSUE THREAD

**Issue by alexei-osipov** [renpy/renpy#1204]
If there any duplicate label names in the game then RenPy fails with error:
`NameError: global name '_menu' is not defined`

This error message does not reflect the problem and does not provided information that might help with the fix (like label name).

Please improve error message.

Part of error log:
```
After initialization, but before game start.
  File "renpy/common/00voice.rpy", line 360, in voice_interact
    if _menu:
NameError: global name '_menu' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "D:\RenpySDK\renpy-6.99.12.4-sdk\renpy\bootstrap.py", line 295, in bootstrap
    renpy.main.main()
  File "D:\RenpySDK\renpy-6.99.12.4-sdk\renpy\main.py", line 430, in main
    renpy.game.script.report_duplicate_labels()
  File "D:\RenpySDK\renpy-6.99.12.4-sdk\renpy\script.py", line 890, in report_duplicate_labels
    if renpy.parser.report_parse_errors():
  File "D:\RenpySDK\renpy-6.99.12.4-sdk\renpy\parser.py", line 2543, in report_parse_errors
    renpy.display.error.report_parse_errors(full_text, error_fn)
  File "D:\RenpySDK\renpy-6.99.12.4-sdk\renpy\display\error.py", line 179, in report_parse_errors
    error_fn=error_fn,
  File "D:\RenpySDK\renpy-6.99.12.4-sdk\renpy\game.py", line 280, in invoke_in_new_context
    return callable(*args, **kwargs)
  File "D:\RenpySDK\renpy-6.99.12.4-sdk\renpy\display\error.py", line 43, in call_exception_screen
    return renpy.ui.interact(mouse="screen", type="screen", suppress_overlay=True, suppress_underlay=True)
  File "D:\RenpySDK\renpy-6.99.12.4-sdk\renpy\ui.py", line 285, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\RenpySDK\renpy-6.99.12.4-sdk\renpy\display\core.py", line 2519, in interact
    i()
  File "renpy/common/00voice.rpy", line 360, in voice_interact
    if _menu:
NameError: global name '_menu' is not defined

Windows-7-6.1.7601-SP1
Ren'Py 6.99.12.4.2187
```
```
Windows-7-6.1.7601-SP1
Ren'Py 6.99.12.4.2187
```

**Comment by alexei-osipov** [comment]
Seems like script was designed to report duplicates but fails to show duplicate label report because of duplicate labels preventing parsing.

**Comment by vollschauer** [comment]
Ren'Py has a pretty "good" error message when it comes to duplicate labels:
```
I'm sorry, but errors were detected in your script. Please correct the
errors listed below, and try again.


The label system_map is defined twice, at
  File "/home/***/system_map.rpyc", line 4 and
  File "/home/***/system_map.rpyc", line 8.

Ren'Py Version: Ren'Py 6.99.13.2461
```

But it looks like Ren'Py 6.99.12.4.2187 is kinda broken, so go with the latest nightly build ....

**Comment by renpytom** [comment]
As mentioned, this has been fixed in 6.99.13. 

Until then, you can fix this by adding:

```
_menu = False
```

to the end of renpy/defaultstore.py
